web_export_view/controllers/controllers.py

```diff
@@ -15,22 +15,22 @@
 class ExcelExportView(ExcelExport):
     def __getattribute__(self, name):
         if name == "fmt":
-            raise AttributeError()
-        return super(ExcelExportView, self).__getattribute__(name)
+            return super(ExcelExportView, self).__getattribute__(name)
+        raise AttributeError()
```

Comment:
Why do you change origin function like this?

Reply:
@yweng8111 It is just for the purpose of improving Test coverage. The logic stays untouched.